aws/resource_aws_ses_domain_mail_from.go

```diff
@@ -0,0 +1,96 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ses"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsSesDomainMailFrom() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsSesDomainMailFromCreate,
+		Read:   resourceAwsSesDomainMailFromRead,
+		Delete: resourceAwsSesDomainMailFromDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"domain": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"mail_from_domain": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
```

Comment:
We should not need `ForceNew: true` here since be able to support updates by calling `SetIdentityMailFromDomain` just like create. 😄